app/views/graphiql/rails/editors/show.html.erb

```diff
@@ -55,13 +55,22 @@
 
     // Defines a GraphQL fetcher using the fetch API.
     var graphQLEndpoint = "<%= graphql_endpoint_path %>";
+    var graphQLMethod = "<%= graphql_endpoint_method %>";
     function graphQLFetcher(graphQLParams) {
-      return fetch(graphQLEndpoint, {
-        method: 'post',
+      var query = graphQLEndpoint;
+      var params = {
+        method: graphQLMethod,
         headers: { 'Content-Type': 'application/json' },
-        body: JSON.stringify(graphQLParams),
-        credentials: 'include',
-      }).then(function (response) {
+        credentials: 'include'
+      };
+
+      if (graphQLMethod.toLowerCase() === 'get') {
+        query += "?query=" + graphQLParams.query;
```

Comment:
Unfortunately `JSON.stringify` won't work as these values are already strings. It will try to "stringify" a string by escaping the quotes, which causes a parser error ([stacktrace](https://gist.github.com/brianstorti/1bec8f9db3da059e187c)). I think we'd need a special handling to add objects in the url anyway.

`encodeURIComponent ` doesn't seem to make a difference, maybe `fetch.js` is already taking care of that, but I can add it if you want.